tasks/publisher.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Include initial OS-specific tasks
+  ansible.builtin.include_tasks: init_{{ ansible_os_family | lower }}.yml
+  vars:
+    _cvmfs_role: publisher
+    _cvmfs_upgrade: "{{ cvmfs_upgrade_server }}"
+
+- name: Include Apache tasks
+  ansible.builtin.include_tasks: apache.yml
+
+- name: Create a directory cvmfs_keys
+  ansible.builtin.file:
+    path: /tmp/keys
+    state: directory
+    mode: '0755'
+
+- name: Copy certificate key on publisher 
+  ansible.builtin.copy:
+    content: "{{ item.crt }}"
+    dest: "{{ item.path | default('/tmp/keys') }}"
+    owner: "{{ item.owner | default('root') }}"
+    group: root
+    mode: "0400"
+  with_items: "{{ cvmfs_certs }}"
+  loop_control:
+    label: "{{ item.path }}"
+
+- name: Create the API key filesystem
+  ansible.builtin.template:
+    src: api-key.gw.j2
+    dest: "{{ item.key_dir | default('/tmp/keys') }}/{{ item.repository }}.gw"
+    owner: "{{ item.owner | default('root') }}"
+    group: root
+    mode: "0644"
+  loop: "{{ cvmfs_repositories_api }}"
```

Comment:
`cvmfs_repositories_api` is a new variable that would need to be added to the README.

Reply:
fixed I just added it and modified the name of this variable I find it more indicative with this name